### PR TITLE
Changes src/main.js to index.js to match docs

### PR DESCRIPTION
**Pull request by tzmanics**
I saw the inconsistency when referring back and forth to the docs and thought this might make it a little clearer.

### CODE REVIEW

File: init/bin.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ const { exit } = require('process')
 const yargs = require('yargs')
 const filterObj = require('filter-obj')
 
-const { init } = require('./main.js')
+const { init } = require('./index.js')
```
Comment (ehmicky) [inline init/bin.js:8]: `init/main.js` and `src/main.js` are different files.

**Review by ehmicky — CHANGES_REQUESTED** [review]
Thanks @tzmanics!
This looks good except for one small issue.

**Review by ehmicky — APPROVED** [review]

**Comment by erezrokah** [comment]
Should we rename the test file https://github.com/netlify/build-plugin-template/blob/master/test/main.js?
I can follow up with a commit to rename it and merge the PR

**Comment by ehmicky** [comment]
Sounds good @erezrokah :+1: 
There is also `init/variables/main.js` and `init/main.js`.

**Comment by erezrokah** [comment]
> There is also `init/variables/main.js` and `init/main.js`.

I thought theses are internal to bootstrapping the project. Anyway we can do those in another PR. Don't want to block this one

**Pull request merged**